### ISSUE TITLE
sorry about the question, but is this library still supported?

### ISSUE THREAD

**Issue by nferruzzi** [stephencelis/SQLite.swift#996]
From the opened PR and issues it doesn't look like it is, unfortunately Ive just discovered this nice lib and I can't decide if its a safe decision to use for a new project or not.
thanks in advance

**Comment by brendand** [comment]
I would love to know this too as I'm considering using it.

**Comment by NathanFallet** [comment]
The last commit is from January, it's not that old. I'm using this library in a lot of iOS apps, and it works perfectly. So I think you should be able to use it without any problem.

**Comment by nferruzzi** [comment]
for me its an hard choice; at the end I used it for a prototype and I had to merge a PR from another user (the one supporting upsert). I got no problems at all and the lib "just worked" for my simple scenario, still I had to commit few fixes to use it with swift packages (If I remember correctly).  But IMHO here are way too many Issues and PR that looks important without a single comment from the author in more than 1 year. I mean is ok, this is a free nice lib but to me it feels like a pity that it doesn't get more love.
